lib/validate.ts

```diff
@@ -269,6 +274,7 @@ export function getValidators(
     if (semver.satisfies(version, '>=1.0.0')) {
         validators = validators.concat(
             new ComponentsValidator(error, componentSet, filePaths),
+            new DirectiveOptionsValidator(error, componentSet),
```

Comment:
Is it safe to add this validator to 1.0+?

Reply:
I would say yes, but could mean a confused customer if they have misconfigured directive options hanging around.